Makefile

```diff
@@ -7,6 +7,7 @@ OPERATOR_IMAGE ?= node-maintenance-operator
 REGISTRY_IMAGE ?= node-maintenance-operator-registry
 
 TARGETS = \
+	cluster-up \
```

Comment:
I assume `pack8s` should also be on this list?

Reply:
removed the symlink from the top level, so there is now no need to add it to the list.